examples/next/pages/examples/listings/ExperienceCard.tsx

```diff
@@ -4,7 +4,7 @@ export const ExperienceCard = ({ img, title }) => (
   <Card
     width="20rem"
     padding="0"
-    borderRadius={theme.radii.large}
+    borderRadius={`${theme.radii.large.value}`}
```

Comment:
This was actually a Typescript thing because `theme.radii.large` is of type `DesignToken` which has a `value` that can be a string or number. So I think this could be changed to `theme.radii.large.value as string`? Also one of my todos was to add more design token types like `RadiusDesignToken` which could be more explicit that its value is a string.